### PR TITLE
Feature/ADT-75 get list of documents from API

### PR DESCRIPTION
**Pull request by BarryGeeDefra**
# **What this PR does / why we need it**:
*A brief description of changes being made*
*Link to the relevant work items: e.g: Relates to ADO Work Item AB#213700 and builds on #3376 (link to ADO Build ID URL)*

# **Special notes for your reviewer**
*Any specific actions or notes on review?*

# Testing
*Any relevant testing information and pipeline runs.*

# Checklist (please delete before completing or setting auto-complete)
- [ ] Story Work items associated (not Tasks)
- [ ] Successful testing run(s) link provided
- [ ] Title pattern should be `{work item number}: {title}`
- [ ] Description covers all the changes in the PR
- [ ] This PR contains documentation
- [ ] This PR contains tests
- [ ] Version number in **package.json** has been updated

# **How does this PR make you feel**:
gif

### CODE REVIEW

File: app/services/documents.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const { format, parseISO } = require('date-fns');
+const { getDocuments } = require('../api/documents')
+
+class DocumentsService {
```
Comment (sfitz42) [inline app/services/documents.js:4]: We don't tend to use classes unless we can help it. E.g. in the rare case we need EventEmitter.

Could you export the functions for a regular module?

File: app/services/documents.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    properties: {
+                        ...item.properties,
+                        createdOn: formattedDate
+                    }
+                }
+            }
+        )
+    }
+
+    async getDocumentsData(id = null) {
```
Comment (sfitz42) [inline app/services/documents.js:25]: Could we separate this into two functions? One for multiple documents and one for a single document.

Just thinking we should only return an array for the multiple documents route. Unless the API bindings already deal with this?

**Review by sfitz42 — CHANGES_REQUESTED** [review]
Looks good! Just a couple things about the DocumentsService :)

File: app/services/documents.js (hunk 2 above)
Comment (BarryGeeDefra) [reply app/services/documents.js]: updated, by id is not required for this ticket. I have updated the code to reflect this.

File: app/services/documents.js (hunk 1 above)
Comment (BarryGeeDefra) [reply app/services/documents.js]: refactored.

**Pull request merged**